.github/workflows/build_scoped_rector.yaml

```diff
@@ -43,7 +43,10 @@ jobs:
                     COMPOSER_TOKEN: ${{ secrets.ACCESS_TOKEN }}
 
             # install only prod dependencies - do not use ramsey, it uses cache including "dev", we want to avoid it here
-            -   run: composer install --no-dev --ansi
+            # somehow psr/http-message needed now when downgrading ssch/typo3-rector
+            -   run: |
+                    composer require psr/http-message
+                    composer update --no-dev --ansi
```

Comment:
`psr/http-message` somehow now required when downgrading ssch/typo3-rector

Reply:
Interesting... This should be probably solved in the typo3 Rector package.

@sabbelasichon Could you check it and remove the package if possible?